packages/cli/src/project/handlebars.ts

```diff
@@ -0,0 +1,33 @@
+import * as changeCase from 'change-case';
```

Comment:
Does this need to be added to the cli package.json?

Reply:
it's already there